server/webapp/WEB-INF/rails.new/webpack/views/shared/header_panel.js.msx

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
any reason for this being a separate file? can we include this as part of `form_helpers`?